lib/api/definitions/TimeToLive.ts

```diff
@@ -15,4 +17,7 @@ import { WithinRangeNumber } from "../../utils/numbers";
 
 export type TimeToLive = WithinRangeNumber<3600, 31536000>;
 
-export const TimeToLive = WithinRangeNumber(3600, 31536000);
+export const TimeToLive = withDefault(
+  WithinRangeNumber(3600, 31536000),
+  3600 as any
```

Comment:
this `as any` bothers me!

Reply:
you can replace `any` with `TimeToLive`, makes more sense and avoids using `any`